lib/decanter/core.rb

```diff
@@ -125,7 +129,7 @@ def unhandled_keys(args)
 
         case strict_mode
         when :ignore
-          p "#{self.name} ignoring unhandled keys: #{unhandled_keys.join(', ')}."
+          p "#{self.name} ignoring unhandled keys: #{unhandled_keys.join(', ')}." if log_unhandled_keys_mode
```

Comment:
What are your thoughts on making this a predicate method?

Reply:
hmm, leaning towards no, since that method is being used more like a reader  for `@log_unhandled_keys_mode`